driver/terraform.go

```diff
@@ -350,7 +337,7 @@ func (tf *Terraform) init(ctx context.Context) error {
 }
 
 // initTask initializes the task
-func (tf *Terraform) initTask() error {
+func (tf *Terraform) initTask(ctx context.Context) error {
```

Comment:
It makes sense to group the workspace init and validate here. The removal of redundancy looks good! But I can't recall if there was prior intention for this -- gonna loop in @lornasong

Reply:
I can't recall any context. I also think adding init and validate here makes sense!